script/generate.py

```diff
@@ -1,123 +0,0 @@
-import argparse
```

Comment:
Maybe we should keep this script during a time but only printing the command that should be used now, in case someone looks to old docs.

Reply:
Ah, that might not be a bad idea.

Reply:
Sorry I didn't meant to keep the script as is, I meant to keep it, but not generating a beat, only printing something about the command that should be used now.

Reply:
OHHHHHHHH